integration-tests/smoke/automation_test.go

```diff
@@ -936,84 +1056,124 @@ func TestAutomationCheckPerformGasLimit(t *testing.T) {
 			t.Parallel()
 			l := logging.GetTestLogger(t)
 			config, err := tc.GetConfig("Smoke", tc.Automation)
-			if err != nil {
-				t.Fatal(err)
-			}
+			require.NoError(t, err, "Failed to get config")
 			a := setupAutomationTestDocker(
 				t, registryVersion, automationDefaultRegistryConfig(config), false, false, &config,
 			)
 
-			consumersPerformance, upkeepIDs := actions.DeployPerformanceConsumers(
+			sb, err := a.ChainClient.Client.BlockNumber(context.Background())
+			require.NoError(t, err, "Failed to get start block")
+
+			consumersPerformance, upkeepIDs := actions_seth.DeployPerformanceConsumers(
 				t,
+				a.ChainClient,
 				a.Registry,
 				a.Registrar,
 				a.LinkToken,
-				a.Deployer,
-				a.ChainClient,
-				defaultAmountOfUpkeeps,
+				1, // It was impossible to investigate, why with multiple outputs it fails ONLY in CI and only for 2.1 and 2.2 versions
```

Comment:
can u explain a little bit what this means?

Reply:
Well, that test originally deployed `defaultAmountOfUpkeeps` number of upkeeps, but then only asserted the behaviour of a single one. I modified it to check behaviour of all upkeeps and it did work on local, but was failing in CI for these two registry versions. For 2.0 it worked. @anirudhwarrier also couldn't figure out why it's failing, so we decided to focus on testing only one contract, like the test did from the beginning, but also to deploy only 1 upkeep.